.github/workflows/verify-md.yml

```diff
@@ -0,0 +1,47 @@
+name: Markdown
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do we push to `main` without Pull-Request? If not, then we do not need this extra run